### DISCUSSION TITLE
LATEX and markdown

### DISCUSSION THREAD

**Discussion by Brybrio** [sr320/course-fish546-2021#54] [Educational Resources]
A while back I was assisting with transcribing some academic textbooks and articles using R Studio, which included complicated formulas with odd signs and symbols. For this I used [LATEX](http://www.docs.is.ed.ac.uk/skills/documents/3722/3722-2014.pdf), which is helpful to preserve organization and had a variety of tools for dealing with unusual characters. I wonder about its potential implementation with Linux and our current tools.

**Comment by sr320** [comment]
Great point!
Yes LATEX is similar to markdown as it is a way of formatting text in a plain text format.
And you are correct - Latex can do much more that markdown, including equations.

Here is nice direct comparison https://github.com/mundimark/markdown-vs-latex

It does appear GitHub will render Latex in Jupyter notebooks.. https://gist.github.com/cyhsutw/d5983d166fb70ff651f027b2aa56ee4e
